examples/experimental/onnx/README.md

```diff
@@ -87,13 +143,23 @@ Please refer to [Docker image build](#docker-image-build) section.
                 -v <MODEL_DIR>:/onnx-models \
                 -v <OUTPUT_DIR>:/output     \
                 onnx_ptq_experimental:latest
-(container) $ ./examples/run_ptq_onnx_models.sh /onnx-models /output $NUMBER_OF_SAMPLES
+(container) $ ./examples/run_ptq_onnx_models.sh [classification|det_and_seg] /onnx-models /output /omz_data $NUMBER_OF_SAMPLES
```

Comment:
is the location ./examples/experimental/onnx/run_ptq_onnx_models.sh instead of ./examples/run_ptq_onnx_models.sh ?

Reply:
Docker image build copies `./examples/experimental/onnx/run_ptq_onnx_models.sh` to `./examples/run_ptq_onnx_models.sh`
https://github.com/openvinotoolkit/nncf/blob/508a44a1cf10d83adde3f48f4039688a6fe760c9/docker/onnx/openvinoep/Dockerfile#L38